extensions/pyRevitTools.extension/checks/cad_audit_check.py

```diff
@@ -21,9 +21,9 @@ def collect_cadinstances(active_view_only):
         cadinstances = DB.FilteredElementCollector(doc, ac_view.Id).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
     else:
         cadinstances = DB.FilteredElementCollector(doc).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
-    
-    if len(cadinstances) >0:
-        return cadinstances, len(cadinstances)
+    cadinstances_len = len(cadinstances)
+    if cadinstances_len >0:
+        return cadinstances, cadinstances_len
```

Comment:
This is not what I meant.
the function should only return cadinstances, then the main function should save the return value into a variable and use `len(variable)` when it prints the length.

Also, `if len(something) > 0:` is not pythonic at all, just `if something:` (this is one of the "unpythonic things here and there" that I was referring to)...

Reply:
gotcha